### PR TITLE
v0.39.0

### PR DESCRIPTION
**Pull request by koszti**
release v0.39.0

### CODE REVIEW

**Comment by judahrand** [comment]
@koszti Would it be possible to get my fixes regarding BigQuery into this release?

**Comment by koszti** [comment]
> @koszti Would it be possible to get my fixes regarding BigQuery into this release? #861

I started CI on the PR, let's see how it goes

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+0.38.1 (2022-02-25)
+-------------------
+- Bump `pipelinewise-tap-kafka` from `4.0.1` to `5.0.0`
+- Bump `pipelinewise-target-bigquery` from `1.1.1` to `1.2.0`
+- Prevent usage of extended transformation feature when FastSync exists
```
Comment (Samira-El) [inline CHANGELOG.md:5]: we also bumped pipelinewise-transform-field to 2.3.0 with new transformations on proeprties of json objects

File: setup.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
     LONG_DESCRIPTION = f.read()
 
 setup(name='pipelinewise',
-      version='0.38.0',
+      version='0.38.1',
```
Comment (Samira-El) [inline setup.py:9]: 3 changes are either breaking changes or new features, we need to bump at least the minor not the patch ... I know we can move to v1 yet so minor it is then 🤷🏽

**Review by Samira-El — APPROVED** [review]

**Pull request merged**